src/wallet/rpcwallet.cpp

```diff
@@ -2650,22 +2653,48 @@ Value zc_raw_pour(const json_spirit::Array& params, bool fHelp)
         throw runtime_error("unsupported pour input/output counts");
     }
 
-    // TODO: #808
-    uint256 pubKeyHash;
+    uint256 joinSplitPubKey;
```

Comment:
I suggested doing it here because it's needed for the call to ``crypto_sign_keypair`` to be locally auditable as memory-safe.